provisioner/powershell/provisioner.go

```diff
@@ -404,17 +403,17 @@ func (p *Provisioner) createFlattenedEnvVars(elevated bool) (flattened string) {
 	envVars["PACKER_BUILDER_TYPE"] = p.config.PackerBuilderType
 
 	// expose ip address variables
-	httpAddr := common.GetHTTPAddr()
-	if httpAddr != "" {
-		envVars["PACKER_HTTP_ADDR"] = httpAddr
+	httpAddr := p.generatedData["PackerHTTPAddr"]
+	if httpAddr != nil && httpAddr != "" {
```

Comment:
This will never be nil, or empty, will it? It'll either be "ERR_HTTP_ADDR_NOT_IMPLEMENTED_BY_BUILDER" or a valid address.